src/core/charpak_mod.F90

```diff
@@ -1,16 +1,8 @@
-!------------------------------------------------------------------------------
-!                  CATChem Model                                              !
-!------------------------------------------------------------------------------
-!BOP
-!
-! !MODULE: charpak_mod.F90
-!
-! !DESCRIPTION: Module CHARPAK\_MOD contains routines from the CHARPAK
-!  string and character manipulation package used by GEOS-Chem.
-!\\
-!\\
-! !INTERFACE:
-!
+!> \file charpak_mod.F90
+!! \brief Module for character string manipulation
+!!
+!!!>
+
```

Comment:
```suggestion
```
Might as well go right up against the module. No special ending tag is needed (i.e. it doesn't seem like `!!!>` does anything special).